.github/workflows/ci.yml

```diff
@@ -18,7 +18,7 @@ jobs:
       tests: ${{ steps.filter.outputs.tests }}
       vendoring: ${{ steps.filter.outputs.vendoring }}
     steps:
-      # For pull requests it's not necessary to checkout the code
```

Comment:
> For pull requests it's not necessary to checkout the code

But in other circumstances than pull requests it seems it is, so let's checkout always for the sake of simplicity